Makefile

```diff
@@ -30,9 +30,17 @@ all-devel: init develop
 	@echo "    make frontend"
 	@echo ""
 
+.PHONY: mini-devel
+# Get minimal dependencies and install Streamlit into Python environment -- but do not build the frontend.
+all-devel: mini-init develop
+
 .PHONY: init
-# Install Python and JS dependencies.
-init: setup pipenv-install react-init scssvars protobuf # react-build release
+# Install all Python and JS dependencies for development.
```

Comment:
We also call `init` when we `make all` so not sure adding `for development` is right here.